lms/djangoapps/commerce/utils.py

```diff
@@ -88,7 +88,7 @@ def payment_page_url(self):
         """
         return self.get_absolute_ecommerce_url(self.config.single_course_checkout_page)
 
-    def get_checkout_page_url(self, *skus):
+    def get_checkout_page_url(self, *skus, **kwargs):
```

Comment:
Just specify `program_uuid=None` here instead of pulling it out below.

Reply:
Tried that. Due to *skus being an iterable, **kwargs needs to hold all keyword arguments.
Proof:
```
def get_checkout_page_url(self, *skus, program_uuid=None):
                                                      ^
SyntaxError: invalid syntax
```

Reply:
Hmm, I think the best thing to do here would be:
```
def get_checkout_page_url(self, program_uuid=None, *skus):
```
But that would precipitate a larger changeset. I leave it to you to decide.

Reply:
I was having issues with that format as well, but that may have been PEBKAC. I think I'll keep it as is and add a docstring reference.